src/ObjCRuntime/INativeObject.cs

```diff
@@ -15,6 +16,13 @@ public interface INativeObject {
 			get;
 		}
 #endif
+
+#if NET
+		// The method will be implemented via custom linker step if the managed static registrar is used
+		// for classes which have an (NativeHandle, bool) or (IntPtr, bool) constructor.
+		[MethodImpl(MethodImplOptions.NoInlining)]
+		static virtual INativeObject? ConstructINativeObject (NativeHandle handle, bool owns) => null;
```

Comment:
This is detected as a breaking change by our tooling.

I see two options:

1. Fix our api comparison tool that detects breaking changes to handle interfaces with default methods (and not consider them breaking).
2. Have the MSR code inject this method in the interface at build time.

Reply:
I made the interface method internal. That seems to be enough for the API comparison tool. The custom linker step changes it to public if MSR is used. I did the same goes for the `INSObjectFactory` interface and that seems to be OK as well.